pkg/lib/scoped/querier.go

```diff
@@ -6,12 +6,18 @@ import (
 
 	"github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
+	meta "k8s.io/apimachinery/pkg/api/meta"
```

Comment:
Nit: nothing pressing, but there's an existing meta package import (metav1) right before before this declaration.